backend/api/src/DB/exams_test.go

```diff
@@ -22,25 +22,29 @@ func TestDatabaseExam(t *testing.T) {
 func TestCreateExam1(t *testing.T) {
 	_, err := helpers.FixtureWrapCreate(t, &helpers.TestExam)
 	assert.Nil(t, err, "After fixture is run a create will be called that shall return no error")
+	_ = helpers.FixtureWrapNonCreate(t)
```

Comment:
Is it neccessary to do for the code?, can be very expensive to run on all tests :/